howso/scikit/scikit.py

```diff
@@ -3,7 +3,8 @@
 import uuid
 
 from howso import engine
-from howso.client import AbstractHowsoClient, HowsoPandasClient
+import howso.client
+from howso.client.base import AbstractHowsoClient
```

Comment:
Are the import changes in this file still relevant? Don't see any cicular depdenecy with this module that would require these changes.

Reply:
Yup, just residual from when I was trying to address that, will revert.

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60